hw6/task1/library.js

```diff
@@ -0,0 +1,194 @@
+const { setImmediate, setInterval } = require('timers');
+
+class Library{    
+    Track = [];
+    Album = [];
+    Artist = [];
+ // ADD METHOD WITH NESTINGS   
+    add(...args){ 
+    //    value.constuctor.name; - return Class name "Track,Artist,Album";
+    //    value.name - return class.name "Linkin park exemple";
+    // IF :for classes! if one value - this CLASS. if more value: first - Class, nexts - parameters this Class;
+    //ELSE: For class parameters. If we have more then one parameters = > [new Class,...parametrs for this class ];
+        let myArguments = [];
+        myArguments.push(...args);        
+                            // TRACK                            
+            if(myArguments[0].constructor.name === "Track"){
```

Comment:
use `instanceof` operator instead of checking the constructor name

Reply:
fixed